### DISCUSSION TITLE
plots not showing

### DISCUSSION THREAD

**Discussion by markpowers70** [Chia-Network/chia-blockchain#17868] [Support]
I cannot see my plots in gui

**Comment by BrandtH22** [comment]
Hey @markpowers70 , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia). 

Generally when plots are not visible I will reboot the computer then if they are still not visible when launching chia try to re-add the plot directories. 

Keep in mind that if using recursive plot scanning and having a failed drive can lead to those plots not being identified properly

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
